data/data/bootstrap/files/usr/local/bin/bootkube.sh.template

```diff
@@ -440,7 +440,10 @@ then
 			--manifests-dir=/assets/manifests \
 			--cloud-credential-operator-image=${CLOUD_CREDENTIAL_OPERATOR_IMAGE}
 
-	cp cco-bootstrap/manifests/* manifests/
+    	# skip copy of manifests in case when CloudCredential capability disabled
+	if [ -d cco-bootstrap/manifests ]; then
+		cp cco-bootstrap/manifests/* manifests/
+	fi
```

Comment:
If I understand correctly: if the CCO capability is disabled when this render command is run, `cco-bootstrap/manifests` will not exist? In which case we need this code to avoid an error due to the nonexistance of the dir?